lwgenerate/azure/test-data/activity-log-with-all-subscriptions.tf

```diff
@@ -0,0 +1,35 @@
+terraform {
+  required_providers {
+    azuread = {
+      source  = "hashicorp/azuread"
+      version = "~> 2.16"
+    }
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "=2.91.0"
```

Comment:
Any reason for us to be explicit with `"=2.91.0"` rather than approximate with `"~> 2.91"`?